### PR TITLE
fix: overflow x bug on dashboard

### PR DESCRIPTION
**Pull request by payn33**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
- This pr fixes an overflow bug on the dashboard page by including an `overflow: hidden` property on the `filter.tsx` container

## Related Tickets & Documents
- Fixes: #677 

### Before 

https://user-images.githubusercontent.com/66219025/209103278-ba0a3c6a-e3e8-4a1e-b173-7c5a987ca8f0.mp4

### After
image

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

File: layouts/filter.tsx [hunk 1]
```diff
@@ -8,14 +8,13 @@ import FilterHeader from "components/molecules/FilterHeader/filter-header";
...
-const FilterLayout = ({children}: {children: React.ReactNode}) => {
-
+const FilterLayout = ({ children }: { children: React.ReactNode }) => {
   const { toolList, selectedTool, filterName, selectedFilter, userOrg } = useNav();
 
   return (
     <>
       <TopNav />
-      <div className="page-container flex min-h-[calc(100vh-(54px+95px))] flex-col items-center">
+      <div className="page-container flex min-h-[calc(100vh-(54px+95px))] flex-col items-center overflow-x-hidden">
```
Comment (0-vortex) [inline layouts/filter.tsx:17]: Thanks for opening a PR for this, however hiding the overflow is not the correct way to solve this problem; we should figure out what the underlying element that breaks the 100% width range and assess whether this is a platform bug.

For reference, can you let us know your operating system and browser so we can replicate a testing environment? 🍕

**Review by 0-vortex — CHANGES_REQUESTED** [review]
Thanks for opening a PR, unfortunately we need some more information to be able to test this on our end! 🍕

File: layouts/filter.tsx (hunk 1 above)
Comment (payn33) [reply layouts/filter.tsx:17]: sure, i'm on windows 11 using a chrome browser, overflow appears around 1300px. following your advice I took a closer look and noticed that changing the tooltip nav.tsx parent elements width property from 100vw to 100% also fixes the overflow without having to hide it.

File: components/organisms/ToolList/nav.tsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ const Nav: React.FC<NavProps> = ({ toolList, selectedTool = "dashboard", selecte
       aria-orientation="horizontal"
       aria-label="Browse the tools"
       tabIndex={0}
-      className="tool-list-nav flex w-[100vw] 2xl:w-full overflow-x-auto overflow-y-hidden gap-2 px-4 md:px-16 bg-light-slate-3 border-b pt-3"
+      className="tool-list-nav flex w-full overflow-x-auto overflow-y-hidden gap-2 px-4 md:px-16 bg-light-slate-3 border-b pt-3"
```
Comment (payn33) [inline components/organisms/ToolList/nav.tsx:25]: not the right fix, removes the horizontal scroll from nav

File: layouts/filter.tsx (hunk 1 above)
Comment (payn33) [reply layouts/filter.tsx]: hi @0-vortex you can review the changes I made?

File: layouts/filter.tsx [hunk 3]
```diff
@@ -8,8 +8,7 @@ import FilterHeader from "components/molecules/FilterHeader/filter-header";
 
 import useNav from "lib/hooks/useNav";
 
-const FilterLayout = ({children}: {children: React.ReactNode}) => {
-
+const FilterLayout = ({ children }: { children: React.ReactNode }) => {
```
Comment (brandonroberts) [inline layouts/filter.tsx:11]: nit: revert unnecessary changes to this file

File: components/organisms/ToolList/nav.tsx [hunk 4]
```diff
@@ -22,7 +22,7 @@ const Nav: React.FC<NavProps> = ({ toolList, selectedTool = "dashboard", selecte
       aria-orientation="horizontal"
       aria-label="Browse the tools"
       tabIndex={0}
-      className="tool-list-nav flex w-[100vw] 2xl:w-full overflow-x-auto overflow-y-hidden gap-2 px-4 md:px-16 bg-light-slate-3 border-b pt-3"
```
Comment (brandonroberts) [inline components/organisms/ToolList/nav.tsx:25]: @SunGoldTech thoughts on this change?

**Review by brandonroberts — CHANGES_REQUESTED** [review]

File: components/organisms/ToolList/nav.tsx (hunk 4 above)
Comment (0-vortex) [reply components/organisms/ToolList/nav.tsx:25]: I'm pro 👍

Comment (SunGoldTech) [reply components/organisms/ToolList/nav.tsx:25]: LGTM... great work @payn33 🍕

**Review by SunGoldTech — APPROVED** [review]

File: layouts/filter.tsx (hunk 3 above)
Comment (payn33) [reply layouts/filter.tsx:11]: this has been done.

File: layouts/filter.tsx [hunk 5]
```diff
@@ -8,15 +8,15 @@ import FilterHeader from "components/molecules/FilterHeader/filter-header";
...
+  
   const { toolList, selectedTool, filterName, selectedFilter, userOrg } = useNav();
 
   return (
     <>
       <TopNav />
       <div className="page-container flex min-h-[calc(100vh-(54px+95px))] flex-col items-center">
-        <div className="info-container min-w-full min-h-[100px]">
+        <div className="info-container w-full min-h-[100px]">
```
Comment (payn33) [inline layouts/filter.tsx:19]: the reason for this change is that a parent element with no fixed width will not allow the children elements with overflow auto or scroll properties to flow properly, causing the parent element to go over the screen.

**Review by brandonroberts — APPROVED** [review]
